doc/usermanual/lighttable/concepts/undoredo.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE sect2 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+               "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd" [
+		<!ENTITY % darktable_dtd SYSTEM "../../dtd/darktable.dtd">
+		%darktable_dtd;
+		]>
+<sect2 status="final" id="lt_undo_redo">
+
+  <title>Undo/redo</title>
+
+    <indexterm>
+      <primary>undo</primary>
+    </indexterm>
+
+    <indexterm>
+      <primary>redo</primary>
+    </indexterm>
+
+    <para>
+      Most changes in the lighttable are recorded and can be
+      reverted. This includes modifications in color labels, ratings,
+      geo-localization, tags, metadata, copy/paste of history or
+      applying a style. Thanks to this recording it is possible to
+      undo changes or redo them to recover a previous state. Note that
+      this undo/redo facility is unlimited in the number of steps
+      while in the lighttable but it is reset each time the view is
+      switched (going to the darkroom for example).
+    </para>
```

Comment:
Just added some punctuation....


Most changes in the lighttable are recorded and can be reverted. This includes modifications to color labels, ratings,       geo-localization, tags, metadata, copy/paste of history, or applying a style.
Thanks to this recording, it is possible to undo changes or redo them to recover a previous state. Note that this undo/redo facility is unlimited in the number of steps while in the lighttable, but it is reset each time the view is switched (going to the darkroom for example).